docs/fr-FR/_summary.md

```diff
@@ -0,0 +1,2 @@
+* [Changements de rupture](fr-FR/breaking_changes.md)
```

Comment:
`Breaking change` do not translate, otherwise it would become a mess.

Reply:
I had written that initially and came back to the non translated version, forgot it in the summary